syncano/models/manager.py

```diff
@@ -849,6 +849,39 @@ def run(self, *args, **kwargs):
         response = self.request()
         return registry.ScriptTrace(**response)
 
+    @clone
+    def get_runtimes(self):
+        """
+        Method which get available runtimes from CORE;
+        :return:
+        """
+        self.method = 'GET'
+        self.endpoint = 'runtimes'
+        self._serialize = False
+        response = self.request()
+
+        class RuntimeChoices(object):
```

Comment:
And if I want to define it outside - probably need to create new file runtimes.py with only this class :) TBH - dunno which is better